.github/workflows/integ.yml

```diff
@@ -4,8 +4,18 @@ on:
   workflow_dispatch:
   push:
     branches: ["main"]
+    paths-ignore:
+      - ".*ignore"
```

Comment:
This is going to be a problem because merging a PR requires the integ workflow to pass.